.github/workflows/test_zfs_root_guide.yml

```diff
@@ -0,0 +1,66 @@
+name: "Test installation guides"
+on:
+  push:
```

Comment:
Let's add path filters to start this workflow only when these root-on-zfs guides were changed https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore

Just to minimize risks, so we'll have working CI even if these tests have any problems.

Reply:
Thanks for the link!  I also wanted to limit the tests to only my guides, however I didn't find the related github action doc page.